tests/test_codecs/test_codecs.py

```diff
@@ -75,26 +77,17 @@ async def test_order(
     data = np.arange(0, 256, dtype="uint16").reshape((32, 8), order=input_order)
     path = "order"
     spath = StorePath(store, path=path)
-    codecs_: list[Codec] = (
-        [
-            ShardingCodec(
-                chunk_shape=(16, 8),
-                codecs=[TransposeCodec(order=order_from_dim(store_order, data.ndim)), BytesCodec()],
-            )
-        ]
-        if with_sharding
-        else [TransposeCodec(order=order_from_dim(store_order, data.ndim)), BytesCodec()]
-    )
 
     with config.set({"array.order": runtime_write_order}):
```

Comment:
should we alter this test to take a `config` kwarg? happy to deem that out of scope for this PR